client/components/common/BaseForm.vue

```diff
@@ -108,4 +113,41 @@ span {
   display: flex;
   align-items: center;
 }
+
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltiptext {
+  visibility: hidden;
+  background-color: var(--darkGray);
+  color: #fff;
```

Comment:
use var(--white) instead in case Olivia changes the variable in her PR

Reply:
sorry what I was meaning was keep what you had on line 124 but switch line 125 to var(--white). Now it is white on white which won't work.